.github/workflows/main.yml

```diff
@@ -83,3 +83,25 @@ jobs:
       run: cross build -vv --target ${{ matrix.platform.target }}
       working-directory: test-crate
       if: ${{ !matrix.platform.test }}
+
+  ios_cross_compile_test:
+    name: Test Cross Compile - ${{ matrix.platform.target }}
+    needs: [ test ]
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - target: aarch64-apple-ios
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        target: ${{ matrix.platform.target }}
```

Comment:
Ah, it seems this was brought by the current config, it's fine then. I'm going to fix it later along with others, so feel free to leave them as-is!

Reply:
Thanks, I just copy/pasted from the existing config for consistency. It's a separate job because I couldn't tell if the `cross` tool would work on macOS from a brief look at it.